packages/portfolio-contract/DESIGN-BETA.md

```diff
@@ -162,6 +162,55 @@ sequenceDiagram
     participant Res as Resolver
   end
 
+  box rgb(163,180,243) Noble
+      participant icaN as icaNoble
+  end
+  box rgb(163,180,243) Axelar
+      participant AX as GMP
+  end
+  box rgb(163,180,243) Arbitrum
+      participant pc as CCTPProxyContract
+      participant acctArb as acctArb*
+      participant aavePos as aavePos*
+  end
+  box rgb(163,180,243) Base
+      participant acctBase as acctBase*
+      participant compPos as compPos*
+  end
+
+  %% get plan
+  UI ->> portfolio: withdraw with steps, <br>steps also include CCTP fee and proxy contract address
+
+  Note over portfolio, aavePos: CCTP back
+  portfolio ->> AX: GMP (withdraw, $2k USDC, Noble ICA address, CCTP fee, CCTP proxy contract address)
+  AX -->> acctArb: withdraw $2k
+  acctArb ->> aavePos: withdraw $2k
+  aavePos ->> acctArb: $2k
+  acctArb ->> pc: depositForBurn($2k)
+  PC -->> icaN: CCTP mint $2k USDC
+  icaN -->> Res: observe mint
+  Res ->> portfolio: ack
```

Comment:
this shows the money getting only as far as the noble ICA.

Are you leaving the IBC transfer back to agoric LCAorch and the localTransfer to the Cash seat implicit on purpose?

Reply:
Good call, updated